apps/frontend/src/app/2_molecules/WalletIdentity/components/DisconnectSubmenu/DisconnectSubmenu.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { FC, PropsWithChildren } from 'react';
+
+import { Menu, MenuItem } from '@sovryn/ui';
+
+export type DisconnectSubmenuProps = {
+  onDisconnect: () => void;
+  address: string;
+  className?: string;
+  dataActionId?: string;
+};
+
+export const DisconnectSubmenu: FC<
+  PropsWithChildren<DisconnectSubmenuProps>
+> = ({ address, onDisconnect, className }) => {
+  return (
+    <Menu className={className}>
+      <MenuItem
+        onClick={async () => {
+          await navigator.clipboard.writeText(address);
+          alert('Address was copied to clipboard.');
+        }}
+        icon={'copy'}
+        iconProps={{ size: 16, className: 'mr-2' }}
+        text="Copy Address"
+      />
+      <MenuItem
+        iconProps={{ size: 16, viewBox: '0 0 16 14', className: 'mr-2' }}
```

Comment:
we already have a 16px size of the icon by default, pls check my comment about the iconProps in iconSvgPaths.ts